OracleAnalytics/patches/6.4-patch/README.md

```diff
@@ -0,0 +1,34 @@
+# Build a Oracle Analytics Server 6.4 Patched Image
+
+This Dockerfile extends the Oracle Analytics Server image (6.4),
+and applies necessary patches.

```

Comment:
Why are these patches necessary?

Reply:
These are the patches which are recommended by OAS Marketplace team to customers and these patches have various critical patches related to security, etc. (across various components like FMW, OPSS, etc.).